src/S3/audio.h

```diff
@@ -24,17 +24,17 @@ int S3OpenSampleDevice(void);
 void S3CloseDevices(void);
 tS3_outlet* S3CreateOutlet(int unk1, int pChannel_count);
 int S3CreateOutletChannels(tS3_outlet* outlet, int pChannel_count);
-void S3DisposeOutlet(tS3_outlet* outlet);
+void S3ReleaseOutlet(tS3_outlet* outlet);
```

Comment:
Asking for rec2, did you invent these names yourself?

Reply:
Largely, yes. Although there are some DRS3* wrapper functions that we do know the names of in the dethrace code, so I used those where possible for names and conventions